docs/apm/traces/get-started-transaction-tracing/set-up-traces-collection-aws-environments.md

```diff
@@ -30,155 +30,107 @@ instances ([Scenario 1 below](#scenario-1-aws-opentelemetry-collector-installati
     ```
-
 1. Execute the command below to create an [AWS CloudFormation](https://aws.amazon.com/cloudformation/) stack that deploys the AWS OpenTelemetry Collector on your cluster:  
-
     ```
     aws cloudformation create-stack --stack-name sumologic-aws-otel-col-ecs-ec2 --template-body file://${TEMPLATE_PATH} --parameters ParameterKey=ClusterName,ParameterValue=${CLUSTER_NAME} --capabilities=CAPABILITY_NAMED_IAM --region=${AWS_REGION}
     ```
-
-1. To check if everything was deployed go to the [CloudFormation Stacks console](https://console.aws.amazon.com/cloudformation/home#/stacks?filteringStatus=active&filteringText=&viewNested=true&hideStacks=false) and check if the **sumologic-aws-otel-col-ecs-ec2** stack status is **CREATE_COMPLETE**.  
-
-    ![stack status.png](/img/traces/stack_status.png)
-
-1. The next step is to check if your deployment is properly running. Go to the [ECS Console](https://console.aws.amazon.com/ecs/home), select the proper region, and select the cluster you used to deploy the AWS OpenTelemetry Collector. Navigate to the **Tasks** tab and check if the task is running.  
-
-    ![task status running.png](/img/traces/task-status-running.png)
-
-1. Finally, click on the task and expand the **Containers** list. In he **Network Bindings > External Link** section, you will find the information on where to send telemetry data.  
-
-    ![external links.png](/img/traces/external-links.png)
+1. To check if everything was deployed go to the [CloudFormation Stacks console](https://console.aws.amazon.com/cloudformation/home#/stacks?filteringStatus=active&filteringText=&viewNested=true&hideStacks=false) and check if the **sumologic-aws-otel-col-ecs-ec2** stack status is **CREATE_COMPLETE**. <br/>  ![stack status.png](/img/traces/stack_status.png)
+1. The next step is to check if your deployment is properly running. Go to the [ECS Console](https://console.aws.amazon.com/ecs/home), select the proper region, and select the cluster you used to deploy the AWS OpenTelemetry Collector. Navigate to the **Tasks** tab and check if the task is running. <br/>  ![task status running.png](/img/traces/task-status-running.png)
+1. Finally, click on the task and expand the **Containers** list. In the **Network Bindings > External Link** section, you'll find the information on where to send telemetry data.<br/>![external links.png](/img/traces/external-links.png)
+  If you plan to send data to the ECS Collector service from a container running in the bridge network mode using the same host, you can use Docker Gateway IP - 172.17.0.1 on EC2 Linux in the application environment variables. For example:
+  ```bash
+  OTEL_EXPORTER_OTLP_ENDPOINT=http://172.17.0.1:55681
+  OTEL_TRACES_EXPORTER=otlp
+  ```
+  However, with this configuration, the collector service task must be running in each of the used EC2 instances.
```

Comment:
@pbrzoska-sumo To clarify, the actual change to this PR was this block of text.

Reply:
ok, if that comes from @mat-rumian (it was not clear from the initial PR), then that's fine as he already approved.